lib/Backend/GlobOpt.cpp

```diff
@@ -1603,13 +1608,15 @@ GlobOpt::OptArguments(IR::Instr *instr)
                     if (builtinFunction == Js::BuiltinFunction::JavascriptFunction_Apply)
                     {
                         CurrentBlockData()->ClearArgumentsSym(src1->AsRegOpnd());
+                        instr->m_func->canOptimizeInlineeArg = true;
```

Comment:
Won't this set to true undo any possible sets to false of `canOptimizeInlineeArg`? For example, if there was a TypeOfElem before ArgOut_A_InlineBuiltIn